orc8r/cloud/deploy/terraform/orc8r-helm-aws/helm.tf

```diff
@@ -13,7 +13,8 @@
 
 # helm tiller service account
 resource "kubernetes_service_account" "tiller" {
-  count = var.existing_tiller_service_account_name == null ? 1 : 0
+  # Deploy tiller only if it does not exist yet and we're in a non-staging environment
+  count = var.existing_tiller_service_account_name == null && !var.orc8r_is_staging_deployment ? 1 : 0
```

Comment:
I think it would be valuable to create an additional layer of indirection, similar to how you're doing it in the helm charts -- `enableLogging` is the variable these modules actually care about. And "is staging deployment" is just 1 way we could decide to disable logging

Reply:
Cool, we had an existing variable to enable tiller, I just used it here.

Though the cost is we have to pay is to include both `orc8r_is_staging_deployment` and `install_tiller` in main.tf instead of just the first one.